kolibri/plugins/learn/assets/src/composables/useProgressTracking.js

```diff
@@ -145,7 +145,7 @@ export default function useProgressTracking(store) {
       const data = response.data;
       set(context, valOrNull(data.context));
       set(complete, valOrNull(data.complete));
-      set(progress_state, threeDecimalPlaceRoundup(valOrNull(data.progress)));
```

Comment:
This rounding was preventing progress very close to 1, but not quite 1 from being incremented by the frontend.